en/setup/dev_env_linux.md

```diff
@@ -91,24 +91,7 @@ Note:
 * ROS is installed with Gazebo7 by default (we have chosen to use the default rather than Gazebo8 or Gazebo9 to simplify ROS development).
 * Your catkin (ROS build system) workspace is created at **~/catkin_ws/**.
 
+<!-- import docs for other tools and next steps. -->
+{% include "_addition_dev_tools.txt" %}
```

Comment:
The last part of the `_addition_dev_tools.txt` is a section next steps, which sends you to build, then you have the Note below. I'm not sure that works. Could we move the Note below into the _addition_dev_tools.txt ?

Reply:
We might need to say "on linux" in the note

Reply:
done

Reply:
I still think the content of `_addition_dev_tools.txt` should not be duplicate in the output but rather apear on its own page and get linked to. Or is that against your concept?

Reply:
Depends on how you expect people to read the docs. I take your point though - pulled out the devtools in #550